system/controllers/admin_controller.py

```diff
@@ -0,0 +1,26 @@
+import tkinter as tk
+from system import views
+from system.database.db_handler import ProductDB
+from system.controllers.abstract_controllers import AbstractAdminController
+
+
+class AdminController(AbstractAdminController):
+    def __init__(self, root, controller):
```

Comment:
Same changes that I suggested to Wasim, rename "controller" to "access_controller" so there is no confusion.
Also, pass in the user that is created in the access_controller.

So overall it should be:
def __ init __ (self, root, user, access_controller):